OJS.Workers.ExecutionStrategies/Python/PythonDjangoOrmExecutionStrategy.cs

```diff
@@ -175,20 +180,20 @@ private void ActivateVirtualEnvironment(IExecutor executor, IExecutionContext<Te
             throw new ArgumentException("Failed to activate virtual environment! " + this.GetErrorOutput(result));
         }
 
-        private void DeleteVirtualEnvironment(IExecutor executor, IExecutionContext<TestsInputModel> executionContext)
+        private void DeleteVirtualEnvironment(IExecutor executor, IExecutionContext<TestsInputModel> executionContext, string envName)
             => this.Execute(
                 PyenvAppFileName,
-                this.ExecutionArguments.Concat(new[] { $"virtualenv-delete {VirtualEnvName}" }),
+                this.ExecutionArguments.Concat(new[] { $"virtualenv-delete {envName}" }),
                 executor,
                 executionContext,
                 executionContext.TimeLimit,
```

Comment:
Let's not use the timeLimit from executionDetails for deleting virtual env, as we have no control over this time.

Reply:
Done